tests/wpt/web-platform-tests/resources/testharness.js

```diff
@@ -921,21 +921,23 @@ policies and contribution forms [3].
 
              var p;
              for (p in actual) {
-                 assert(expected.hasOwnProperty(p), "assert_object_equals", description,
+                 assert(p in expected, "assert_object_equals", description,
```

Comment:
Is this ES6 syntax? If so, I'm inclined to use ES5 syntax since not all browsers support ES6.